libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -543,11 +543,7 @@ void AP_SerialManager::init()
 
 #ifndef HAL_BUILD_AP_PERIPH
                 case SerialProtocol_RCIN:
-                    if (AP::RC().has_uart()) {
-                        GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "Duplicate RCIN configured on SERIAL%u", i);
-                    } else {
-                        AP::RC().add_uart(uart);
-                    }
+                    AP::RC().add_uart(uart);
```

Comment:
```suggestion
                    if (!AP::RC().has_uart()) {
                        AP::RC().add_uart(uart);
                    }
```
Just to maintain current behaviour.

Reply:
right